### PR TITLE
Custom parameter filter

### PR DESCRIPTION
**Pull request by bazelajoanna**
Changes in this PR:

- added under basic parameters add “see more” link for desktop devices
- added additional modal with all custom parameters (mobile and desktop devices)
- added all the logic connected with setting sensors 

[Trello card ](https://trello.com/c/SAIoqXwW/1847-session-filter-both-parameter-picker-custom)

### CODE REVIEW

File: app/javascript/react/store/sensorsSlice.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+const initialState: SensorsState = {
+  sensors: [],
+  fetchSensorsStatus: StatusEnum.Idle,
+};
+
+export const fetchSensors = createAsyncThunk(
+  "sensors",
+  async (sessionType: SessionType, { rejectWithValue }) => {
+    const sessionTypeCapitalized =
+      sessionType[0].toUpperCase() + sessionType.slice(1);
```
Comment (JoannaThelema) [inline app/javascript/react/store/sensorsSlice.ts:30]: Why do you need to do this? What does `.slice(1)` do? 😅

File: app/javascript/react/utils/setSensor.ts [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    const firstSensor = allSensors[0];
+
+    return firstSensor;
+  };
+
+  if (sessionType === SessionTypes.FIXED) {
+    switch (selectedParameter) {
+      case FixedBasicParameterTypes.PARTICULATE_MATTER:
+        return {
+          sensorName: "Government-PM2.5",
```
Comment (JoannaThelema) [inline app/javascript/react/utils/setSensor.ts:29]: We have `"Government-PM2.5"` hardcoded in a few places. I would extract this to const and reuse

**Review by JoannaThelema — APPROVED** [review]
Only some minor comments. Great job! 🎉

File: app/javascript/react/store/sensorsSlice.ts (hunk 1 above)
Comment (evulins) [reply app/javascript/react/store/sensorsSlice.ts:30]: I think this is to ensure that when the sessionType is passed to enpoint it’s in the correct format.

File: app/javascript/react/types/sensors.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  PARTICULATE_MATTER: {
+    AIRBEAM_PM25: "AirBeam-PM2.5",
+    GOVERNMENT_PM25: "Government-PM2.5",
+  },
+  HUMIDITY: "AirBeam-RH",
+  NITROGEN_DIOXIDE: "Government-NO2",
+  OZONE: "Government-Ozone",
+  TEMPERATURE: "AirBeam-F",
+  SOUND_LEVEL: "Phone microphone",
+};
```
Comment (JoannaThelema) [inline app/javascript/react/types/sensors.ts:18]: ❤️

File: app/javascript/react/types/filters.ts [hunk 4]
```diff
@@ -38,11 +38,16 @@ type ParameterType =
   | typeof FixedBasicParameterTypes[keyof typeof FixedBasicParameterTypes]
   | typeof MobileBasicParameterTypes[keyof typeof MobileBasicParameterTypes];
 
+const UnitSymbols = {
+  ParticulateMatter: "µg/m³",
+};
```
Comment (JoannaThelema) [inline app/javascript/react/types/filters.ts:43]: ❤️

**Pull request merged**